tests/base/src/test/java/org/wildfly/security/auth/FileSystemSecurityRealmTest.java

```diff
@@ -102,6 +126,21 @@ public void testCreateIdentityWithLevels() throws Exception {
         identity.dispose();
     }
 
+    @Test
+    public void testCreateIdentityWithLevelsEncryption() throws Exception {
+        FileSystemSecurityRealm securityRealm = FileSystemSecurityRealm.builder()
+                .setRoot(getRootPath(false))
```

Comment:
Just to check, is this test making use of an existing path or is a new one being created?

Reply:
Also, is there a test that attempts to parse an encrypted identity file (i.e., an explicit test that makes use of an existing filesystem path where the identity file has schema version 1.1 and "enc_base64" configured)?

Reply:
This is a new path, removed false from `getRootPath()`

Added a test `testVerifyCredentialsPreExistingIdentity()` to check an existed encrypted identity file